src/main/resources/application.properties

```diff
@@ -1,81 +1,68 @@
 base.directory=/srv/blackduck
 install.dry.run=true
-
```

Comment:
Why did you delete all these spaces?

Reply:
I wonder if my save actions in the IDE deleted them.  I will update the file to have the spacing again.